### ISSUE TITLE
Interpretation of the strength of platform token WA - you gave me the cold shoulder yesterday, you will be out of my league today

### ISSUE THREAD

**Issue by TONGZHENGSHIJIE** [TONGZHENGSHIJIE/-#108]
No one thought that the market, which had been in a downturn for more than a year, platform token was the first to see the trend of recovery. The rise in several weeks is a rare good news for more than a year.
In fact, last year, I had recommended the WA platform token for everyone.
The person who started early, smiled; the person who hesitated for a long time, wants to cry without tears. WA relies on strength to interpret everyone - I love you yesterday, I can't afford to climb today.
Choosing a token to eliminate the factors of our own funds, we must first consider the platform risk and the value of the token.
Platform risk factor

Nowadays, the black platform of the universal investment is everywhere, attracting investors' investment eyes with attractive conditions. Once it falls into the black platform of investment, it is the beginning of losses. Therefore, before investing, we must do a good job of platform inspection.

WA is the platform token issued by the Wafcoin token trading platform. Was registered in the Virgin Islands in 2018, Wafcoin is the world's first token trading platform with dual licenses, and its operations and  transactions have legal status.

Headquartered in Australia, Wafcoin Operations and Management Center has independent operation teams in Germany, Estonia, Hong Kong, Malaysia, Singapore, Philippines, India and other countries and regions. It is a professional token trading service provider facing the world. Based on the fourth-generation blockchain technology and the fast and smooth user experience, the Wafcoin platform is dedicated to creating an innovative community global token trading platform with an extremely fast, safe, free, self-government, high-efficiency, transparent services for each user, to become a community-based organization that is truly common, co-governance and sharing.
Wafcoin is a reliable and credible investment platform.
The value factor of token

The purpose of the investment is to rely on the principal to obtain income, so it is important to choose a valuable token.

As a platform token, WA has its own technology and advantages: optimized for complex off-the-counter purchases, users can quickly and fast carry out trading operations; enter the market with zero capital, no capital risk to get profit and other advantages, to avoid many risks for users, to stabilize the security of assets; WA's global issuance will never increase, helping it to have a very high investment value.

Secondly, in the future development of WA, the project team pays more attention to the multi-area, multi-scenario and landing application of WA, which further enhances the investment value of WA. The platform has reached a deep strategic cooperation with Newbuy Mall, the world's first cross-border e-commerce provider that supports token settlement. All the tokens in the WA and the platform can be consumed in the mall, which constantly stimulates the value of WA; In the application of WA, the developers have also focused on the tourism and real estate industries, and started to improve the industry's pain points in the tourism and real estate industries with the most advanced fourth-generation blockchain technology, and accelerate the landing application; in the payment function of WA, the team cooperated with many business companies around the world to continuously expand the field of payment scenarios.

In summary, WA is a token with advantages worth to start with.

For more information, please visit the official website of wafcoin: www.wafcoin.net
